### DISCUSSION TITLE
Tensorflow ModuleNotFoundError

### DISCUSSION THREAD

**Discussion by daniel-black** [antonior92/automatic-ecg-diagnosis#18] [Q&A]
Hi @antonior92, 

I'm running into this ModuleNotFoundError when I try to run `python model.py`:

![image](https://user-images.githubusercontent.com/53156789/155035124-2af22486-316b-45bc-ab06-789bee304f9e.png)

The requirements.txt file specifies `tensorflow=2.2` which I believe needs to have a double equals `==` to install. When I make the change to `==` and run `pip install -r requirements.txt`, I can see that tensorflow2.2 installs just fine. 

Would you be able to share the exact version numbers of the packages that you have running on your machine?

Thanks!

**Comment by antonior92** [comment]
You are right: there was a mistake on the requirements file. Indeed it should be double equals `==`. I just fixed it tough...

Did you have problems when running with `tensorflow==2.2`?

**Reply by daniel-black** [reply]
Yeah I was. I figured out my issue though -- my laptop does not have a cuda enabled Nvidia GPU. Google Colab seems to be a good alternative though.
